.github/workflows/update-db-csv-files.yml

```diff
@@ -0,0 +1,70 @@
+name: Update DB files
+on:
+  schedule:
+    - cron: "0 0 * * *"
```

Comment:
Consider adding a comment that provides a human-readable description of the cron value.